jquery.ellipsis.js

```diff
@@ -210,13 +210,19 @@
 
     // only bind to window resize if required
     if (base.opts.responsive) {
+      var windowWidth = $(window).width();
 
       /**
        * resize() resets necessary vars
        * and content and then re-initialises
        * the Ellipsis script
        */
       var resize = function() {
+        if ($(window).width() == windowWidth) {
+          return;
+        }
+        windowWidth = $(window).width();
+
```

Comment:
@taberrr, thanks for this fix. It would be great if you could cache the `window` reference and add a comment explaining why this is required though please if you don't mind?

```javascript
if (base.opts.responsive) {
  var $window = $(window);
  var windowWidth = $window.width();

  var resize = function() {
    // Scrolling on iOS6+ fires the resize event. This ensures the width has actually changed
    // before continuing. https://tinyurl.com/ios-scroll-resize
    if ($window.width() === windowWidth) {
      return;
    }
    
    windowWidth = $window.width();
    //etc.
  }

  $window.on('resize.' + namespace, resize);
}
```